packages/devtools_app/test/memory/allocation_tracing/allocation_profile_tracing_view_test.dart

```diff
@@ -0,0 +1,221 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:devtools_app/devtools_app.dart';
```

Comment:
import libraries directly

Reply:
I wish there was a way to prevent VSCode from auto-importing the root project library... is there any reason we export everything from `package:devtools_app/devtools_app.dart` anyway?

Reply:
I agree - is there a way to prohibit a specific import pattern in your IDE? @DanTup 

package:devtools_test imports this file. We also have g3 custom screens that import devtools_app.dart to use shared functionality.

Reply:
> I agree - is there a way to prohibit a specific import pattern in your IDE? @DanTup

I'm not sure I understand the request here. Is the issue that the names could be imported from multiple files (eg. because of `exports`)? If so, the code completion should show each class name for each file that could be imported, and you can select which one you want:

<img width="875" alt="Screenshot 2022-08-17 at 17 31 09" src="https://user-images.githubusercontent.com/1078012/185193765-72d6ac7a-5510-4ff3-b157-00f62087abf7.png">
<img width="866" alt="Screenshot 2022-08-17 at 17 31 14" src="https://user-images.githubusercontent.com/1078012/185193771-a99cec0d-0535-455c-b3f2-206e41b813b8.png">

I don't know of any way to set preferences around these though. Perhaps if the rules could be captured in a lint, then it could also be used to exclude some patterns from the completion/quick fixes?